api/turing/middleware/authorization.go

```diff
@@ -104,7 +104,7 @@ func getResourceFromPath(path string, prefix string) string {
 	// if a user has READ/WRITE permissions on /projects/{project_id}, they would also have the same
 	// permissions on all its sub-resources. Thus, trimming the resource identifier to aid quicker
 	// authz matching and to efficiently make use of the in-memory authz cache, if enabled.
-	parts := strings.Split(strings.TrimPrefix(path, "/"), "/")
+	parts := strings.Split(strings.TrimPrefix(strings.TrimPrefix(path, prefix), "/"), "/")
```

Comment:
I thought this change was intentional 👀

Reply:
No, unfortunately, I had overlooked this. Turing stores the path prefix `/v1` which must be removed from the resource prior to authz checks. I had accidentally removed this trimming.